BGAnimations/ScreenGameplay underlay/PerPlayer/UpperNPSGraph.lua

```diff
@@ -60,9 +61,8 @@ return Def.ActorFrame{
 				:queuecommand("Update")
 		end,
 		UpdateCommand=function(self)
-			song_percent = scale( GAMESTATE:GetCurMusicSeconds(), 0, last_second, 0, width )
-			-- song_percent can be negative but we don't want to draw this Quad during that time, so use math.max()
-			self:zoomtowidth( math.max(song_percent, 0) ):sleep(0.25):queuecommand("Update")
+			song_percent = scale( GAMESTATE:GetCurMusicSeconds(), first_second, last_second, 0, width )
+			self:zoomtowidth(clamp(song_percent, 0, width)):sleep(0.25):queuecommand("Update")
```

Comment:
I'm not sure whether `song_percent` can be negative anymore... so I ended up removing the comment but left the relevant code. I think using `clamp` is a bit more clear than `math.max`, so I also ended up changing to that.